pom.xml

```diff
@@ -330,7 +328,7 @@
                                 </wait>
                             </run>
                             <build>
-                                <from>${informix.image}:14.10.FC9W1DE</from>
+                                <from>icr.io/informix/informix-developer-database:14.10.FC9W1DE</from>
```

Comment:
There is a different registry used here. Is it intentional? If yes then maybe a new Maven property should be introduced that will hold the registry name. And the `from` will be constructed frem registry name property and image name property

Reply:
Yes, the latest Informix 14.10 image is in the IBM Container Registry under the 'informix' label while the 12.10 images are with the older 14.10 images in Docker Hub, under the 'ibmcom' label. 
I'll sew something up...